### PR TITLE
Multiple call-back handler.

### PR DESCRIPTION
**Pull request by templarfelix**
Dispatch multiples callback handlers for RequestMessage.

### CODE REVIEW

File: org.hive2hive.core/src/main/java/org/hive2hive/core/network/messages/request/IRequestMessage.java [hunk 1]
```diff
@@ -25,11 +26,19 @@
...
+	 */
+	public void addCallBackHandler(IResponseCallBackHandler handler);
+
+	/**
 	 * Getter
 	 * 
 	 * @return the callback handler (if set)
 	 */
-	public IResponseCallBackHandler getCallBackHandler();
+	public Set<IResponseCallBackHandler> getCallBackHandler();
```
Comment (nicoruti) [inline org.hive2hive.core/src/main/java/org/hive2hive/core/network/messages/request/IRequestMessage.java]: Please rename to getCallBackHandler**s**

File: org.hive2hive.core/src/main/java/org/hive2hive/core/network/messages/request/RoutedRequestMessage.java [hunk 2]
```diff
@@ -16,18 +18,22 @@
...
 
-	private transient IResponseCallBackHandler handler;
+	private transient Set<IResponseCallBackHandler> handler = new HashSet<>();
 
 	public RoutedRequestMessage(String targetKey) {
 		super(targetKey);
 	}
 
-	public final IResponseCallBackHandler getCallBackHandler() {
+	public final Set<IResponseCallBackHandler> getCallBackHandler() {
```
Comment (nicoruti) [inline org.hive2hive.core/src/main/java/org/hive2hive/core/network/messages/request/RoutedRequestMessage.java]: Please rename to getCallBackHandlers

File: org.hive2hive.core/src/main/java/org/hive2hive/core/network/messages/request/RoutedRequestMessage.java [hunk 3]
```diff
@@ -16,18 +18,22 @@
...
 		return handler;
 	}
 
 	public final void setCallBackHandler(IResponseCallBackHandler handler) {
```
Comment (nicoruti) [inline org.hive2hive.core/src/main/java/org/hive2hive/core/network/messages/request/RoutedRequestMessage.java]: Method can now be removed (or accept a list of handlers)

**Comment by nicoruti** [comment]
Thanks for your improvements! Looks good to me.
I added some comments. Formatting has become little weird, but I can do that before merging.

**Comment by templarfelix** [comment]
Changes apply.

**Comment by nicoruti** [comment]
Thanks for your work!

**Pull request merged**